### ISSUE TITLE
recordedit: refactoring app.js to use Reference 

### ISSUE THREAD

**Issue by hongsudt** [informatics-isi-edu/chaise#596]

**Comment by jrchudy** [comment]
This issue is still pending on merging `recordedit2-master` into Chaise. We need to finish updating the e2e tests first before we do that.

**Comment by jrchudy** [comment]
The `recordedit2-master` branch has been merged into master. Closing this issue... finally.
